.npmignore

```diff
@@ -1,15 +1,13 @@
 *
+packages/dd-trace/src/external-logger/test/index.spec.js
 !ext/**/*
 !packages/*/lib/**/*
 !packages/*/src/**/*
 !packages/*/index.js
-!scripts/**/*
+!scripts/preinstall.js
```

Comment:
Should this actually be removed? Preinstall scripts are discouraged and can trip security tools.

Reply:
Possibly... Worth having a follow up discussion.

Reply:
It seems to just be doing what the `engines` field already does...so not really sure what reason there is to keep it. Was there some issue with the `engines` field being disregarded with some package managers or something?